src/Device/Driver/LX_EOS/LXEosDevice.hpp

```diff
@@ -16,6 +16,85 @@
 
 using std::string_view_literals::operator""sv;
 
+struct EosDeclarationStruct
```

Comment:
This should have a `static_assert` checking the size of the struct at compile time to verify its correctness